test/unit/spec/narrative_core/kbaseNarrativeDataList-spec.js

```diff
@@ -4,11 +4,187 @@
 /*global beforeEach, afterEach*/
 /*jslint white: true*/
 define([
-    'kbaseNarrativeDataList'
-], function(Widget) {
-    describe('Test the kbaseNarrativeDataList widget', function() {
-        it('Should do things', function() {
+    'kbaseNarrativeDataList',
+    'jquery',
+    'narrativeConfig',
+    'kb_service/client/workspace',
+    'base/js/namespace'
+], function (DataList, $, Config, Workspace, Jupyter) {
+    'use strict';
+    var $dataList = $('<div>');
+    var dataListObj = null;
 
+    const fakeNSUrl = 'https://ci.kbase.us/services/fake_url';
+    const narrativeServiceInfo = {
+        version: '1.1',
+        id: '12345',
+        result: [{
+            git_commit_hash: 'foo',
+            hash: 'bar',
+            health: 'healthy',
+            module_name: 'SampleService',
+            url: fakeNSUrl
+        }]
+    };
+
+    function mockServiceWizard() {
+        jasmine.Ajax.stubRequest(Config.url('service_wizard')).andReturn({
+            status: 200,
+            statusText: 'HTTP/1/1 200 OK',
+            contentType: 'application/json',
+            responseText: JSON.stringify(narrativeServiceInfo)
+        });
+    };
```

Comment:
Codacy's being whiny about this semicolon on line 37. My bad, that. Would you mind removing?